.travis.yml

```diff
@@ -1,22 +1,14 @@
 language: php
 
 php:
-  - 5.5
   - 5.6
   - 7.0
   - 7.1
-  - hhvm
-
-matrix:
-  include:
-    - php: 5.5
-      env: dependencies=lowest
 
 sudo: false
 
 before_script:
   - composer install -n
-  - if [ "$dependencies" = "lowest" ]; then composer update --prefer-lowest --prefer-stable -n; fi;
 
 script:
   - phpunit
```

Comment:
Regarding the failing test I think it's not the same version of PHPUnit that is used in both cases, could you try replacing this line with `vendor/bin/phpunit` (so it's the PHPUnit of Composer and not the one of Travis CI that is used).